### PR TITLE
Haar 3306 approved premised template

### PR DESCRIPTION
**Pull request by indyv-moj**

### CODE REVIEW

File: src/main/resources/templates/template_hmpps-approved-premises-api.mustache [hunk 1]
```diff
@@ -0,0 +1,755 @@
...
+                {{#cpp-details}}
+                    <table class="data-table">
+                        <tr>
+                            <td class="data-column-25">Name</td>
+                            <td class="data-column-25">Probation region</td>
+                            <td class="data-column-25">Telephone</td>
+                            <td class="data-column-25">Email</td>
+                        </tr>
+                        <tr>
+                            <td>{{ optionalValue (formatDate name) }}</td>
```
Comment (simon-mitchell) [inline src/main/resources/templates/template_hmpps-approved-premises-api.mustache:157]: are these correct with formatDate?

Comment (indyv-moj) [reply src/main/resources/templates/template_hmpps-approved-premises-api.mustache:157]: good spot, blaming my bad copy and paste skills 😄

Comment (indyv-moj) [reply src/main/resources/templates/template_hmpps-approved-premises-api.mustache]: Removed the unnecessary formatDate calls

**Review by simon-mitchell — APPROVED** [review]
LGTM

**Pull request merged**